Dockerfile.cpu

```diff
@@ -26,35 +30,22 @@ RUN echo 'export PATH=/opt/conda/bin:$PATH' > /etc/profile.d/conda.sh && \
     wget --quiet https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O ~/miniconda.sh && \
     /bin/bash ~/miniconda.sh -b -p /opt/conda && \
     rm ~/miniconda.sh
-ENV PATH /opt/conda/bin:$PATH
-
-# Setup AllenNLP
-WORKDIR /stage
 
+# Create a conda environment so we have a reproducible version of Python
```

Comment:
Why are you installing conda at all? Docker is a self-contained environment already - you can just `pip install` stuff locally without needing the virtual env while using an appropriate python base image.